Python_basic/Lists.py

```diff
@@ -0,0 +1,42 @@
+# create list of 100 random numbers from 0 to 1000
+# sort list from min to max (without using sort())
+# calculate average for even and odd numbers
+# print both average result in console
+# Each line of code should be commented with description.
+
+# Commit script to git repository and provide link as home task result.
+
+
+# import 'random' library for random numbers
+import random
+
+# creating a list of 100 random numbers from 0 to 1000
+rand_list = random.sample(range(0, 1000), 100)
+
+# an empy list for a sorted list
+sorted_list = []
+odd_numbers = []
+even_numbers = []
+
+# while rand_list has the numbers execute next iteration
+while rand_list:
+    # searching the min value from rand_list and append sorted list with this value
+    sorted_list.append(min(rand_list))
+    # deleting the min value from rand_list
+    rand_list.remove(min(rand_list))
```

Comment:
Pleas check sort algorythms, it also looks bad. No need to change it for this homework though.